app/qml/map/MMMapController.qml

```diff
@@ -1183,6 +1188,11 @@ Item {
       }
 
       case "view": {
+        if ( identifyHighlight.geometry !== null )
```

Comment:
Can you add an explanation comment here please?

Reply:
sure